.github/workflows/add-template.yml

```
@@ -113,19 +127,23 @@ jobs:
     steps:
       - uses: peter-evans/close-issue@v1
         with:
-          comment: ":white_check_mark: Congratulations! Your template has been verified."
+          comment: ":white_check_mark: Congratulations! Your template has been verified and added to App Builder Template Registry."
           token: ${{ secrets.GITHUB_TOKEN }}
   fail-validation-comment:
     name: Validation failed
     runs-on: ubuntu-latest
-    needs: [check-npm-package, check-github-link, check-npm-link]
+    needs: [check-npm-package, check-github-link, check-npm-link, add-to-registry]
     if: ${{ always() && (needs.check-npm-package.result != 'success' || needs.check-github-link.outputs.error != '') }}
```

Hi, I noticed that `needs.add-to-registry.outputs.error` was missing from the if-statement. If check-npm-package and check-github-link complete successfully but add-to-registry has an error, it won't get printed. 
```suggestion
    if: ${{ always() && (needs.check-npm-package.result != 'success' || needs.check-github-link.outputs.error != '' || needs.add-to-registry.outputs.error != '') }}
```

---

@AmyJZhao , good catch! But I'll propose to replace
`|| needs.add-to-registry.outputs.error != ''` with `|| needs.add-to-registry.result != 'success'`.
The "check-github-link" job is _independent_ from other jobs as we would like to run it every time in parallel with checking the npm link/nmp package, it has `continue-on-error: true`. So it will always have the "success" status, and that is why we check the error output. The "add-to-registry" job has _hard_ dependencies, it doesn't contain `continue-on-error: true`, so it will fail, succeed or will be skipped.

---

^^^ Then it looks like we don't need `needs.check-npm-package.result != 'success'` as `needs.add-to-registry.result != 'success'` covers the case when "check-npm-package" fails. "add-to-registry" will have the "skipped" status then.

---

@AmyJZhao thanks! Fixed.
@slitviachenko applied your suggestion.